package.json

```diff
@@ -1,86 +1,79 @@
 {
-  "name": "npm-outdated",
-  "displayName": "npm Outdated",
-  "description": "Displays a diagnostic message in package.json files for packages which have newer versions available.",
-  "version": "0.0.0-semantically-released",
-  "publisher": "mskelton",
-  "author": "Mark Skelton",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mskelton/vscode-npm-outdated.git"
-  },
-  "bugs": {
-    "url": "https://github.com/mskelton/vscode-npm-outdated/issues"
-  },
-  "homepage": "https://github.com/mskelton/vscode-npm-outdated#readme",
-  "license": "ISC",
-  "keywords": [
-    "npm",
-    "vscode-extension",
-    "vscode"
-  ],
-  "eslintConfig": {
-    "extends": [
-      "@mskelton",
-      "@mskelton/eslint-config/typescript"
-    ]
-  },
-  "scripts": {
-    "lint": "eslint .",
-    "ts": "tsc",
-    "build": "webpack",
-    "start": "webpack --watch",
-    "vscode:prepublish": "npm run build"
-  },
-  "icon": "assets/icon.png",
-  "engines": {
-    "vscode": "^1.49.0"
-  },
-  "categories": [
-    "Linters",
-    "Other"
-  ],
-  "activationEvents": [
-    "onLanguage:json"
-  ],
-  "main": "./out/extension.js",
-  "devDependencies": {
-    "@babel/core": "^7.11.6",
-    "@mskelton/eslint-config": "^5.11.3",
-    "@types/node": "^14.10.1",
-    "@types/semver": "^7.3.3",
-    "@types/vscode": "^1.49.0",
-    "@typescript-eslint/eslint-plugin": "^3.10.1",
-    "eslint": "^7.9.0",
-    "eslint-plugin-sort": "^1.5.0",
-    "prettier": "^2.1.1",
-    "semantic-release": "^17.1.1",
-    "semantic-release-vsce": "^3.0.1",
-    "ts-loader": "^8.0.3",
-    "typescript": "^4.0.2",
-    "vsce": "^1.79.5",
-    "webpack": "^4.44.1",
-    "webpack-cli": "^3.3.12"
-  },
-  "dependencies": {
-    "semver": "^7.3.2"
-  },
-  "contributes": {
-    "configuration": {
-      "type": "object",
-      "title": "npm Outdated",
-      "properties": {
-        "npm-outdated.level": {
-          "type": "string",
-          "default": "patch",
-          "enum": [
-            "major",
-            "minor",
-            "patch"
-          ],
-          "markdownDescription": "Minimum semver bump required for a package to display as outdated. See npm's [Semver docs](https://docs.npmjs.com/cli/v6/using-npm/semver) for more details."
+    "name": "npm-outdated",
```

Comment:
4 space formatting?  Did you setup Prettier?

Reply:
Ah, my prettier is 4 spaced, I will make it 2 spaces on this project and I will recommit all files.